crmsh/ui_node.py

```diff
@@ -458,24 +518,21 @@ def _call_delnode(self, node):
             if rc != 0:
                 logger.error('"%s" failed, rc=%d, %s', cmd, rc, err)
                 return False
+        if utils.ext_cmd(cls.node_delete % node) != 0 or \
+                utils.ext_cmd(cls.node_delete_status % node) != 0:
+            logger.error("%s removed from membership, but not from CIB!", node)
+            return False
         return True
 
     @command.completers(compl.nodes)
     def do_delete(self, context, node):
         'usage: delete <node>'
-        if not utils.is_name_sane(node):
-            return False
-        if not xmlutil.is_our_node(node):
-            logger.error("node %s not found in the CIB", node)
-            return False
-        if not self._call_delnode(node):
-            return False
-        if utils.ext_cmd(self.node_delete % node) != 0 or \
-                utils.ext_cmd(self.node_delete_status % node) != 0:
-            logger.error("%s removed from membership, but not from CIB!", node)
-            return False
-        logger.info("node %s deleted", node)
-        return True
+        logger.warning('`crm node delete` is deprecated and will be dropped in the near future. It is auto-replaced as `crm cluster remove -c {}`.'.format(node))
```

Comment:
@zzhou1 And please review this warning expression

Reply:
I'm okay with that. Or, if you want the the tone to be softer a little, I could suggest 

```suggestion
        logger.warning('`crm node delete` is deprecated and will very likely be dropped in the near future. It is auto-replaced as `crm cluster remove -c {}`.'.format(node))
```